cloud/scope/powervs_cluster.go

```diff
@@ -879,7 +879,7 @@ func (s *PowerVSClusterScope) isDHCPServerActive() (bool, error) {
 		return false, err
 	}
 
-	requeue, err := s.checkDHCPServerStatus(dhcpServer.Status)
+	requeue, err := s.checkDHCPServerStatus(*dhcpServer)
```

Comment:
How about logging the name here before calling the function? So we can avoid passing whole object.